internal/server/api.yaml

```diff
@@ -97,9 +97,31 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/ComposeResponse'
-
+        '400':
+          description: the compose request is malformed
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/HTTPErrorList'
```

Comment:
Why is this a list of errors? Seems like you only ever put one into it, no?

Reply:
Yup, but it has to comply with https://jsonapi.org/format/#errors:

> Error objects MUST be returned as an array keyed by errors in the top level of a JSON:API document.

Reply:
Maybe I should rename the HTTPErrorList to just HTTPErrors? because it's an object container 'errors: []'

Reply:
Ah, interesting. Thanks for the link.

Both `ErrorList` and `Errors` work for me. This name is not exposed as API.